@ecomplus/storefront-components/src/html/QuantitySelector.html

```diff
@@ -0,0 +1,49 @@
+<div class="col col-md-4">
+  <div 
+    class="py-2" 
+    v-if="title"
+  >
+    <span>
+      {{ title }}
+    </span>
+  </div>
+
+  <div 
+    class="quantity-selector" 
+    v-for="item in items"
+  >
+    <input
+      type="number"
+      class="form-control"
+      v-on:change="change"
+      v-model.number="selectedQuantities[item._id]"
+      :min="item.min_quantity"
+      :max="selectedQuantities[item._id] >= item.quantity ? 0 : selectedQuantities[item._id] + quantityRemaining"
```

Comment:
Ainda não concordo com isso aqui hahaha
Desta forma se eu selecionar 10 e o item tiver 10 em estoque você vai setar `max=0` e o navegador pode até mostrar um erro no input se validado, não?
Além disto se o estoque do produto for 1 e o máximo do kit for 100, se não tiver nenhum dele selecionado, o `max` aqui não era pra ser 100, mas sim 1, na sua implementação este `max` começaria como 100 e usuário conseguiria setar 100 de uma vez se quisesse...
Acho que o certo é algo como:

```suggestion
      :max="Math.min(item.quantity, selectedQuantities[item._id] + quantityRemaining)"
```